build.sbt

```diff
@@ -7,7 +7,7 @@ ThisBuild / sbtVersion := "1.6.1"
 ThisBuild / scalaVersion := "2.13.6"
 ThisBuild / organization := "systems.informal"
 
-ThisBuild / apalacheVersion := "#unstable"
+ThisBuild / apalacheVersion := "#main"
 ThisBuild / benchmarksToolVersion := apalacheEnableVersion.value
```

Comment:
Not sure how `apalacheEnableVersion` works. Is this a new command that only builds Apalache but doesn't run any benchmarks. If so, can you update the docs?

Reply:
This is not a new command, but has always been used by the build system under the hood.

Documented here: d87fa297a03a4fa421a132d8a856521ce6ca469c